api_schema.graphql

```diff
@@ -149,7 +150,6 @@ type Mutation {
     repoURL: String
     initSwitch: Boolean!
     authOption: String!
-    sshKeyPath: String
```

Comment:
use the latest file from source repo and submit a new commit. Looks like your change has removed this entry